larq/quantizers.py

```diff
@@ -248,7 +248,7 @@ def get_config(self):
         }
 
 
-ste_tern = SteTern
+ste_tern = utils.register_alias(SteTern, "ste_tern")
 
```

Comment:
I don't really like that ste_tern is an alias for SteTern. It is different in usage from ste_sign but intuitively it should be the same. My proposal is that ste_tern is some default ternarizer such as SteTern(). Anyways this might be another issue again and I can change that later.

Reply:
Good point. I will change this tomorrow, since this requires some more changes.